### PR TITLE
Adding clang-config file and its changes

### PR DESCRIPTION
**Pull request by bremanandjk**
#### Describe Bug or Feature

Apply clang formatter across PBS.

### CODE REVIEW

**Comment by bremanandjk** [comment]
Could you plz review the changes @bhroam & @kjakkali

File: src/include/pbs_internal.h [hunk 1]
```diff
@@ -255,6 +256,8 @@ struct pbs_config
 
 extern struct pbs_config pbs_conf;
 
+// clang-format off
+
```
Comment (bhroam) [inline src/include/pbs_internal.h:256]: Should this be clang-format on?

File: src/include/basil.h [hunk 2]
```diff
@@ -976,3 +977,5 @@ typedef struct basil_response {
 #define BASIL_ERR_TRANSIENT	0x00000001UL
 
 #endif /* _BASIL_H */
+
+// clang-format off
```
Comment (bhroam) [inline src/include/basil.h:980]: Shouldn't this be clang-format on?  It is actually needed since it is the end of the file?

**Review by bhroam — COMMENTED** [review]
I didn't review the actual formatting changes.  I just reviewed the 1st and 3rd commit.

**Comment by bremanandjk** [comment]
> I didn't review the actual formatting changes. I just reviewed the 1st and 3rd commit.

Thanks for comments, Addressed them.

**Review by bhroam — APPROVED** [review]

**Review by kjakkali — APPROVED** [review]
I did review pipeline script changes and it looks good to me.

**Comment by bremanandjk** [comment]
Thanks @bhroam & @kjakkali for your Approval, Please take this PR forward.

**Pull request merged**